benchmark.py

```diff
@@ -44,8 +36,27 @@
 parser.add_argument('--data-parallel', action = 'store_true')
 parser.add_argument('--backward', action = 'store_true')
 parser.add_argument('--input-time-dim-multiple', type = int, default = 128)
+
+parser.add_argument('--output', type=str)
```

Comment:
Please change to `parser.add_argument('--output-path', '-o')` like we do in all other places, we also don't specify the implicit `type=str`

Reply:
done